db/migrate/20161101133602_add_currency_column_to_payment_settings.rb

```diff
@@ -0,0 +1,16 @@
+class AddCurrencyColumnToPaymentSettings < ActiveRecord::Migration
+  def up
+    add_column :payment_settings, :minimum_price_currency, :text, after: :minimum_price_cents
+    add_column :payment_settings, :minimum_transaction_fee_currency, :text, after: :minimum_transaction_fee_cents
+    sql = "UPDATE communities c, payment_settings s
+           SET s.minimum_price_currency = c.available_currencies,
+               s.minimum_transaction_fee_currency = c.available_currencies
```

Comment:
Currently there seems to be one marketplace in the production DB where `available_currencies` holds a list of multiple currencies that would be migrated to the `payment_settings` table in this migration. Is that a problem?